addons/web/static/tests/views/kanban/kanban_view.test.js

```diff
@@ -6134,6 +6134,48 @@ test("quick create column with x_name as _rec_name", async () => {
     expect(".o_kanban_group").toHaveCount(3, { message: "should now have three columns" });
 });
 
+test.tags("desktop")("count should be hidden in kanban folded stage if zero", async () => {
+    await mountView({
+        type: "kanban",
+        resModel: "partner",
+        arch: `
+            <kanban on_create="quick_create">
+                <field name="bar"/>
+                <templates>
+                    <t t-name="kanban-box">
```

Comment:
Hello @adch-odoo I think we can use `kanban-card` As we currently simplify the kanban. And also remove the `<div>`.

Thanks